glade/task_properties.glade

```diff
@@ -348,41 +348,11 @@
 	      </child>
 
 	      <child>
-		<widget class="GtkOptionMenu" id="billstatus menu">
+		<widget class="GtkComboBox" id="billstatus menu">
 		  <property name="visible">True</property>
 		  <property name="tooltip" translatable="yes">Is this task ready to be billed to the customer? &quot;Hold&quot; means maybe, but not yet, needs review.   &quot;Bill&quot; means print this on the next invoice.   &quot;Paid&quot; means that it should no longer be included on invoices.</property>
-		  <property name="can_focus">True</property>
-		  <property name="history">0</property>
-
-		  <child internal-child="menu">
-		    <widget class="GtkMenu" id="convertwidget12">
-		      <property name="visible">True</property>
-
-		      <child>
-			<widget class="GtkMenuItem" id="convertwidget13">
-			  <property name="visible">True</property>
-			  <property name="label" translatable="yes">Hold</property>
-			  <property name="use_underline">True</property>
-			</widget>
-		      </child>
-
-		      <child>
-			<widget class="GtkMenuItem" id="convertwidget14">
-			  <property name="visible">True</property>
-			  <property name="label" translatable="yes">Bill</property>
-			  <property name="use_underline">True</property>
-			</widget>
-		      </child>
-
-		      <child>
-			<widget class="GtkMenuItem" id="convertwidget15">
-			  <property name="visible">True</property>
-			  <property name="label" translatable="yes">Paid</property>
-			  <property name="use_underline">True</property>
-			</widget>
-		      </child>
-		    </widget>
-		  </child>
+		  <property name="can_focus">False</property>
```

Comment:
Why this change? Doesn't the `GtkComboBox`  have useful actions on focus?

Reply:
Keyboard navigation works correctly with can_focus false. Set it to true, and you cannot tab away from it. Nobody really knows why and the documentation appears useless on the subject but it's been noted (https://gitlab.gnome.org/GNOME/gtk/-/issues/2077).